### ISSUE TITLE
Missing types for `<use href={"str"} />`

### ISSUE THREAD

**Issue by lucacasonato** [preactjs/preact#4601]
- [x] Check if updating to the latest Preact version resolves the issue

**Describe the bug**

As of 10.25.0, the SVG `use` tag types are missing the `href` attribute. This is however a support attribute for `use` according to MDN: https://developer.mozilla.org/en-US/docs/Web/SVG/Attribute/href

**Comment by lucacasonato** [comment]
Oh I could have sworn I checked this - sorry. Let me try again.

**Comment by lucacasonato** [comment]
Yeah, that solved it. Somehow my update only updated to 1.25.0, not 1.25.1 - oh well. Sorry for the noise :)

**Comment by rschristian** [comment]
No worries, happens to all of us :)

We appreciate you opening an issue anyhow! 10.25 was a big types change for us, please let us know if you see anything else out of the ordinary.
